erpnext/support/doctype/issue/issue.py

```diff
@@ -391,3 +398,32 @@ def get_holidays(holiday_list_name):
 	holiday_list = frappe.get_cached_doc("Holiday List", holiday_list_name)
 	holidays = [holiday.holiday_date for holiday in holiday_list.holidays]
 	return holidays
+
+
+@frappe.whitelist()
+def create_new_task_from_issue(self, assign_to):
```

Comment:
```suggestion
def create_new_task_from_issue(issue, assign_to):
```

`self` is a convention for objects to refer to self. It shouldn't be used like this.